applications/mne_scan/libs/scDisp/realtimeevokedsetwidget.cpp

```diff
@@ -167,6 +167,10 @@ RealTimeEvokedSetWidget::RealTimeEvokedSetWidget(QSharedPointer<RealTimeEvokedSe
 
     //set layouts
     this->setLayout(m_pRTESetLayout);
+
+    //Initialize leftover scalars to default values
+    m_iMaxFilterTapSize = m_pRTESet->getValue()->evoked.first().data.cols();
```

Comment:
m_pRTESet can be empty. check if this is the case before you access the data. 
`if(m_pRTESet->getValue()->evoked.isEmpty()) {...}`